docs/tasks/semester-start-end-tasks.qmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "DataHub Semester Start and End Tasks"
+---
+
+This document outlines the tasks for preparing DataHub for the start of a semester and for concluding semester activities.
+
+## Semester Start Tasks
+
+### 1. Setup and Configuration
+- [ ] **Update JupyterHub version**: Check for and apply updates to [Z2JH](https://z2jh.jupyter.org/en/latest/changelog.html) JupyterHub (if required)
+- [ ] **Update JupyterLab/Notebook version**: Check for and apply updates to JupyterLab and Notebook (if required)
```

Comment:
this and the previous step should be done during a maintenance window, so most likely these belong in 'end of semester' tasks

Reply:
Okay, I have seen us do it during the start of the semester but I get your larger point around the ideal workflow

Reply:
well, we pretty much do this during every maintenance window...  and since it's almost never 100% smooth, it usually takes us to the beginning of the semester to get it done.  :)

Reply:
eg:  it's better to start this process before the semester starts